llama-index-core/llama_index/core/indices/multi_modal/base.py

```diff
@@ -127,7 +125,7 @@ def is_image_vector_store_empty(self) -> bool:
     def is_text_vector_store_empty(self) -> bool:
         return self._is_text_vector_store_empty
 
-    def as_retriever(self, **kwargs: Any) -> BaseRetriever:
```

Comment:
why remove the return type? It returns a base retriever?

Reply:
Right, but if you run mypy on this file, it will complain that its missing a return type

There needs to be a return type -- if not base retriever, we should be marking the exact return type then

Reply:
Alright, I have added the return type annotation. However, to avoid duplicating the import, I have edited some other files so that the inner import can be removed without running into circular import errors.